common/src/api/external/error.rs

```diff
@@ -255,32 +255,66 @@ impl From<Error> for HttpError {
     }
 }
 
-impl<T: std::fmt::Debug> From<progenitor::progenitor_client::Error<T>>
+pub trait ClientError: std::fmt::Debug {
+    fn message(&self) -> String;
+}
+
+// TODO this From may give us a shortcut in situtations where we want more
```

Comment:
```suggestion
// TODO this `From` may give us a shortcut in situtations where we want more
```

(sorry for the nit, it just threw me off while reading it)